flink-autoscaler-plugin-jdbc/src/test/java/org/apache/flink/autoscaler/jdbc/event/AbstractJdbcAutoscalerEventHandlerITCase.java

```diff
@@ -430,6 +447,75 @@ void testScalingEventWithParallelismChange() throws Exception {
                         });
     }
 
+    @MethodSource("getExpiredEventHandlersCaseMatrix")
+    @ParameterizedTest
+    void testCleanExpiredEvents(int expiredRecordsNum, Duration eventHandlerTtl) throws Exception {
+
+        final Clock clock = Clock.systemDefaultZone();
+        initExpiredEventHandlerRecords(eventHandlerTtl, expiredRecordsNum, clock);
+        final Timestamp createTimeOfUnExpiredRecord = Timestamp.from(clock.instant());
+        jdbcEventInteractor.createEvent(
```

Comment:
Why don't insert event via event handler? It's a little wired that creating event hanldler after inserting data. In addion, It could cover more cases.